docs/installation.rst

```diff
@@ -2,11 +2,180 @@
 Installation
 ============
 
-At the command line::
+Install
+=======
 
-    $ easy_install puente
+Activate your virtual environment, then do:
 
-Or, if you have virtualenvwrapper installed::
+.. code-block:: bash
 
-    $ mkvirtualenv puente
-    $ pip install puente
+  $ pip install puente
+
+
+Configure
+=========
+
+Puente configuration goes in the ``PUENTE`` setting in your Django settings
+file. Here's a minimal example::
+
+   PUENTE = {
+       'ROOT': BASE_DIR,
```

Comment:
If you start a new Django project, it gets defined in the `settings.py` file. So I suspect someone who starts Django projects would know what this is.

I'll switch it to `'/path/to/project/root'`.